docs/src/main/sphinx/admin/spill.rst

```diff
@@ -15,8 +15,8 @@ implemented on the application level to address specific needs of Trino.
 
 Properties related to spilling are described in :doc:`properties-spilling`.
 
-Memory management and spill
----------------------------
+Resource management and spill
```

Comment:
This one should still say "Memory" since spilling only happens to memory usage - not all "resources" in general.